src/helpers/index.js

```diff
@@ -17,4 +17,14 @@ const getUserSecureStorage = async () => {
   return { username, token };
 };
 
-export { setUserSecureStorage, deleteUserSecureStorage, getUserSecureStorage };
+const getTokenSecureStorage = () => {
+  const token = SecureStore.getItemAsync('token');
```

Comment:
should not it be async? may be could be done like
const getTokenSecureStorage = async () => {
  return await SecureStore.getItemAsync('token');
};
since u assign anyway token later

Reply:
so when i tested it worked without async here and then just using async when im calling this function so i left it like this. should i change? and ok i will just use return here. makes sense.

Reply:
i think it mean to be called as async await since it is async process https://docs.expo.io/versions/latest/sdk/securestore/